src/drivers/distance_sensor/tfmini/TFMINI.cpp

```diff
@@ -224,7 +224,7 @@ void
 TFMINI::start()
 {
 	// schedule a cycle to start things
-	ScheduleOnInterval(100_us);
+	ScheduleOnInterval(2_ms);
```

Comment:
Even this seems excessive. Wondering if it was supposed to be 100_ms in the first place?

Reply:
Agreed. Reduced to 7ms.